spec/factories/admin_users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :admin_user do
+    email { [Faker::Internet.email].sample }
```

Comment:
I tried putting in this way `sequence :email { |n| "#{Faker::Internet.email} #{n}" }` but give an error,  related to what you are talking about. I will try as you commented.